### DISCUSSION TITLE
runing tests failed

### DISCUSSION THREAD

**Discussion by jxj-ward** [idaholab/moose#20158] [Q&A Getting Started] [answered]
Dear all.

I just started learning moose. When I installed moose to the step "./run_tests -j4 "according to tutorial, it showed that some tests about "fvkernels" failed. I would like to know how to solve this situation.

Thanks 

![image](https://user-images.githubusercontent.com/53478986/151497099-919071da-a995-4bae-afa9-70a2a8b493c9.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

Do these tests pass when ran individually? (go in test/tests/fvkernels/mms and run the test there, or in test/tests/fvkernels/one-var-diffusion actually, that's a simpler test)

And you should have more information higher in the run_tests log, could you please paste it for some of the tests here?

Guillaume
